src/vulkan-renderer/imgui.cpp

```diff
@@ -41,34 +37,12 @@ ImGUIOverlay::ImGUIOverlay(const wrapper::Device &device, const wrapper::Swapcha
     ImGuiIO &io = ImGui::GetIO();
     io.FontGlobalScale = m_scale;
 
-    spdlog::debug("Loading ImGUI vertex shader");
-
-    m_vert_shader = std::make_unique<wrapper::Shader>(m_device, VK_SHADER_STAGE_VERTEX_BIT, "ImGUI vertex shader",
+    spdlog::debug("Loading ImGUI shaders");
+    m_vertex_shader = std::make_unique<wrapper::Shader>(m_device, VK_SHADER_STAGE_VERTEX_BIT, "ImGUI vertex shader",
                                                       "shaders/ui.vert.spv");
-
-    VkPipelineShaderStageCreateInfo shader_info{};
-    shader_info.sType = VK_STRUCTURE_TYPE_PIPELINE_SHADER_STAGE_CREATE_INFO;
-    shader_info.stage = VK_SHADER_STAGE_VERTEX_BIT;
-    shader_info.module = m_vert_shader->module();
-    shader_info.pName = m_vert_shader->entry_point().c_str();
-
-    m_shaders.push_back(shader_info);
-
-    spdlog::debug("Loading ImGUI fragment shader");
-
-    m_frag_shader = std::make_unique<wrapper::Shader>(m_device, VK_SHADER_STAGE_FRAGMENT_BIT, "ImGUI fragment shader",
+    m_fragment_shader = std::make_unique<wrapper::Shader>(m_device, VK_SHADER_STAGE_FRAGMENT_BIT, "ImGUI fragment shader",
```

Comment:
Wow the imgui wrapper didn't even use the shader wrapper? That was outdated!

Reply:
It did, but now the pipeline shader bit is handled by the render graph :partying_face: